test/csuite/timestamp_abort/smoke.sh

```diff
@@ -27,10 +27,12 @@ fi
 $TEST_WRAPPER $test_bin $default_test_args
 $TEST_WRAPPER $test_bin $default_test_args -c
 #$TEST_WRAPPER $test_bin $default_test_args -L
+$TEST_WRAPPER $test_bin $default_test_args -B -I 3
 $TEST_WRAPPER $test_bin -m $default_test_args
 $TEST_WRAPPER $test_bin -m $default_test_args -c
 #$TEST_WRAPPER $test_bin -m $default_test_args -L
 $TEST_WRAPPER $test_bin -C $default_test_args
 $TEST_WRAPPER $test_bin -C $default_test_args -c
+$TEST_WRAPPER $test_bin -C $default_test_args -B -I 3
```

Comment:
Given that we are finding issues at the moment, I'm not sure we want to add these to general testing yet.

Reply:
In fact, your PR testing failed running this!

Reply:
If there are no objections, I'll check this in with these two lines commented out. In that way, we can easily put them back in later.